src/DynamoCoreWpf/ViewModels/Core/AnnotationViewModel.cs

```diff
@@ -175,6 +175,19 @@ public Double FontSize
             }
         }
 
+        [JsonIgnore]
+        public Guid GroupStyle
```

Comment:
Maybe name it with `id` so we know it is not a style?

Reply:
Done.